src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes.
Better make Random class field and initialize it using constructor.

Reply:
Sorry I am quite confused, could you explain it in other way?

Reply:
The Random class will be created each time we call this method. That is unnecessary.
Need move this class to the class field, and create once in the constuctor.
Like:
```
public class ColorSupplier {
    private Random random;
    
    public ColorSupplier() {
        random = new Random();
    }
    
    public Color getRandomColor() {
        int index = random.nextInt(Colors.values().length);
}
```